zoo/board_games/mcts_bot.py

```diff
@@ -150,12 +202,20 @@ def rollout(self):
 
     def backpropagate(self, result):
```

Comment:
"""
    This method performs backpropagation from the current node.

    It increments the number of times the node has been visited and the number of wins for the result.
    If the current node has a parent, the method recursively backpropagates the result to the parent.
    """